specification/azsadmin/resource-manager/network/Microsoft.Network.Admin/2015-06-15/Network.json

```diff
@@ -0,0 +1,283 @@
+            "get": {
+                "x-ms-examples": {
+                    "Get overview of the state of the network resource provider": {
+                        "$ref": "./examples/ResourceProviderState/Get.json"
+                    }
+                },
+                "tags": [
+                    "Network"
+                ],
+                "description": "Get an overview of the state of the network resource provider.",
+                "operationId": "ResourceProviderState_Get",
+                "parameters": [
+                    {
+                        "$ref": "Network.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "Network.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/AdminOverview"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Resource": {
+            "description": "Base Resource Object",
+            "type": "object",
+            "properties": {
+                "id": {
+                    "description": "URI of the resource.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "name": {
+                    "description": "Name of the resource.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "type": {
+                    "description": "Type of resource.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "location": {
+                    "description": "Region location of resource.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "tags": {
+                    "description": "List of key value pairs.",
+                    "type": "object",
+                    "additionalProperties": {
+                        "type": "string"
+                    }
+                }
+            },
+            "x-ms-azure-resource": true
+        },
+        "ResourceTenant": {
+            "description": "Base Resource Tenant Object",
+            "type": "object",
+            "properties": {
+                "provisioningState": {
+                    "description": "The provisioning state.",
+                    "type": "string"
+                },
+                "subscriptionId": {
+                    "description": "The subscription ID.",
+                    "type": "string"
+                },
+                "tenantResourceUri": {
+                    "description": "The tenant resource URI.",
+                    "type": "string"
+                }
+            }
```

Comment:
Are there any properties of the `ResourceTenant` that can be marked as readonly ? seems provisioningState can be readonly right?